### ISSUE TITLE
Merge snippets and contrib repos.

### ISSUE THREAD

**Issue by tbnorth** [leo-editor/leo-editor#532]
Seems like https://github.com/leo-editor/snippets and https://github.com/leo-editor/leo-editor-contrib have the same intent.  `snippets` has been used more, but `contrib` is better described / named maybe?  I think one should be merged into the other than the one being dropped should just have a README.md pointing to the other.

**Comment by tbnorth** [comment]
Hmm, pffft, now I look at contrib I'm not sure it's that similar, maybe it would be sufficient to update its readme to indicate it's archival and point people to snippets - opinions?

**Comment by edreamleo** [comment]
The best links for most people might be on Leo's home page. Snippets are already there.  I'll add a contrib link.  I am wary of anything more fancy.

**Comment by tbnorth** [comment]
Hmm, not sure how valuable contrib. is really - should have reviewed it more before starting this issue.  It *looks* like a place to put things like the "all features test file" @vitalije mentioned before, but probably isn't.  I think this could be closed and at most contrib's readme could be updated to suggest it's archival and point to snippets.

**Comment by edreamleo** [comment]
I'll close this issue now.
